new_core/screens/initial.js

```diff
@@ -40,18 +49,24 @@ export function update () {
   green = parseInt(colorTween.green.value, 10)
   blue = parseInt(colorTween.blue.value, 10)
 
+  // This is to finish testing the dynamic Json functionality
+  /*if(json.getFile(0) !== undefined){
+    alert(json.getJson(0).select)
+  }//*/
+
 }
 
 export function render (canvas, ctx) {
-  ctx.fillStyle = "rgb(" + red + "," + green + "," + blue + ")"
 
   // Added to get rid of artifacts from dead pictures
   ctx.clearRect(0, 0, canvas.width, canvas.height);
 
+  ctx.fillStyle = "rgb(" + red + "," + green + "," + blue + ")"
   ctx.fillRect(0,0,canvas.width,canvas.height)
 
   // This draws the image onto the screen
 	ctx.drawImage(jslix.getImg(0), 90, 120, 200, 20)
   //ctx.drawImage(jslix.quickImage("../image/mobile/cwttitle.png"), 10, 120, 200, 20)
 
+  fps.display(ctx, 4, 10)
```

Comment:
when we move this line into screen.js after line 19, then all screens can show the fps :)

Reply:
Yeah, that is a very good move. I think merging this with the performance.js file would be a decent move, but I think just closing these conversations and starting fresh would be a good move considering the high amount of changes to get this in line.